### DISCUSSION TITLE
Possibly a new version of drawArc

### DISCUSSION THREAD

**Discussion by DaveMoss69** [skia4delphi/skia4delphi#274] [Ideas]
Hi would it be possible to have a new DrawArc function in the future?  
That would be similar to drawcircle, but with a start and end angle.
something like:
DrawArc(arcCenterX, arcCenterY, radius, startAngle, endAngle paint);

Thanks
Dave

**Comment by viniciusfbb** [comment]
Hello @DaveMoss69, thank you for the suggestion!

I also prefer your overload, however, in other previous discussions we decided that we will not add/change anything that is not mandatory at the api level, because we always need to keep it as close to the original Skia so that updates to the official Skia are not so laborious for us (we are already 1 year out of date because there is already a lot of manual work, a task that need some weeks of work). Furthermore, always keeping the API as close to the original has the benefit that the documentation and examples of one serve the other. https://api.skia.org/classSkCanvas.html#ab7d43d64e7e6d756e60a6d42f3dc472d

**Reply by DaveMoss69** [reply]
That makes sense, thanks for the prompt reply once again.
